src/main/resources/log4j2.fabric.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" packages="com.mojang.util">
+	<Appenders>
+
+		<!--	System out	-->
+		<Console name="SysOut" target="SYSTEM_OUT">
+			<PatternLayout pattern="[%d{HH:mm:ss}] [%t/%level]: %msg%n" />
+		</Console>
+
+		<!--	Vanilla server gui	-->
+		<Queue name="ServerGuiConsole">
+			<PatternLayout pattern="[%d{HH:mm:ss} %level]: %msg%n" />
+		</Queue>
+
+		<!--	latest.log same as vanilla	-->
+		<RollingRandomAccessFile name="LatestFile" fileName="logs/latest.log" filePattern="logs/%d{yyyy-MM-dd}-%i.log.gz">
+			<PatternLayout pattern="[%d{HH:mm:ss}] [%t/%level]: %msg%n" />
+			<Policies>
+				<TimeBasedTriggeringPolicy />
+				<OnStartupTriggeringPolicy />
+			</Policies>
+		</RollingRandomAccessFile>
+
+		<!--	Debug log file	-->
+		<RollingRandomAccessFile name="DebugFile" fileName="logs/debug/debug.log" filePattern="logs/debug/%d{yyyy-MM-dd}-%i.log.gz">
+			<PatternLayout pattern="[%d{HH:mm:ss}] [%t/%level]: %msg%n" />
+
+			<!--	Keep 3 files max	-->
+			<DefaultRolloverStrategy max="3" fileIndex="min"/>
+			
+			<Policies>
+				<TimeBasedTriggeringPolicy />
+				<OnStartupTriggeringPolicy />
+			</Policies>
+
+		</RollingRandomAccessFile>
+
+	</Appenders>
+	<Loggers>
+		<Root level="debug">
```

Comment:
This root level of debug will be effective at runtime and trigger some unintended vanilla side effects, such as activating debug input streams, registry validation, and a few other behaviors. Consider setting this to `info` for regular gameplay purpose and `trace` for mod development purpose.

Reply:
See updated commit